pkg/tractusx/types.go

```diff
@@ -40,8 +40,9 @@ type QualityGuideline interface {
 // QualityResult implements test result via Passed bool and in case of error a
 // ErrorDescription.
 type QualityResult struct {
-	Passed           bool
-	ErrorDescription string
+	Passed              bool
+	ErrorDescription    string
+	ErrorDescriptionWeb string
```

Comment:
Do we need two different descriptions at the same time? I would think, that a caller only needs a single error description. With the suggested option to configure the output via global constant, a caller can still influence what will be returned as ErrorDescription